packages/office-ui-fabric-react/src/components/SearchBox/examples/SearchBox.CustomIcon.Example.tsx

```diff
@@ -2,15 +2,13 @@ import * as React from 'react';
 import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
 
 // tslint:disable:jsx-no-lambda
-export class SearchBoxCustomIconExample extends React.Component<any, any> {
-  public render(): JSX.Element {
-    return (
-      <SearchBox
-        placeholder="Filter"
-        onFocus={() => console.log('onFocus called')}
-        onBlur={() => console.log('onBlur called')}
-        iconProps={{ iconName: 'Filter' }}
-      />
-    );
-  }
-}
+export const SearchBoxCustomIconExample = () => {
```

Comment:
This can be simplified, since there is no logic in the example:

```tsx
export const SearchBoxCustomIconExample = () => (
  <SearchBox ... />
);
```